modules/build/src/main/scala/scala/build/options/BuildOptions.scala

```diff
@@ -157,7 +158,18 @@ final case class BuildOptions(
           implicit val ec = finalCache.ec
           finalCache.logger.use {
             val path = javaHomeManager.get(jvmId).unsafeRun()
-            os.Path(path)
+            // can throw if path is empty or otherwise invalid
+            try os.Path(path)
```

Comment:
Rather than a try-catch, we can make `os.Path` less likely to throw by calling `os.Path(path, Os.pwd)`.

Reply:
Thanks @alexarchambault for reviewing the PR.

I don't doubt there's a much better work-around -- the issue in this case was that the 'JAVA_HOME' environment variable wasn't set (nor supplied as a param to scala-cli).

The upshot was a colleague was trying to use my scala-cli script and seeing a very obscure-looking error message:

``` requirement failed:  is not an absolute path```

This PR is an attempt to simply approve on that so other people don't have to pull down the source to figure out why their scripts are mysteriously failing on different environments:


![image](https://user-images.githubusercontent.com/181305/144442768-a2b02719-ed46-418b-b9a7-233bc61466ee.png)

Reply:
It's your world - I just live in it (and am a big fan already of scala-cli). I would see this not as an "either/or", but a "both/and".

If any future work-arounds do indeed work, then discussions about the 'catch' clause are moot ... it's just a bit of code noise.

If there *is* an environment where that line (which comes from a 3rd party lib) fails, then this just makes life that much nicer for the end users

Reply:
Hello @alexarchambault - Just wanted to see if this could be added based on the above rationale?